personalize.module

```diff
@@ -1129,30 +1129,33 @@ function personalize_verify_agent($agent_name, $show_messages = TRUE) {
  * @param stdClass $agent_data
  *   (Optional) When provided then agent-specific transition steps can be
  *   included.  This is used for the "Scheduled" status.
+ * @param bool $include_all
+ *   (Optional) Include all transitions that are available.  If FALSE, only
+ *   those transitions that should be shown outside the campaign details
+ *   are returned.
```

Comment:
So what we're using this for is on the campaign list page Malini doesn't want transitions shown that make a not editable campaign editable, e.g., going from complete to paused, running to paused, scheduled to paused or not started.  She wants the user to go to the campaign details page for that.  On the campaign details page (at least when acquia_lift isn't enabled) we need to provide those transitions so that there is somewhere to make that change.  Do you think explaining this better would help?